datumaro/plugins/cvat_format/extractor.py

```diff
@@ -315,6 +316,17 @@ def _load_items(self, parsed):
         return parsed
 
 class CvatImporter(Importer):
+    @classmethod
+    def detect(cls, context: FormatDetectionContext) -> None:
+        annot_file = context.require_file('*.xml')
+
+        with context.probe_text_file(
+            annot_file, "must be an XML file with an \"annotations\" root element",
+        ) as f:
+            _, root_elem = next(ElementTree.iterparse(f, events=('start',)))
+            if root_elem.tag != 'annotations':
```

Comment:
Looks like the same test would work for labelme.

Reply:
Unfortunately, no, because it would be insufficient to distinguish it from VOC.